src/antsibull/cli/antsibull_build.py

```diff
@@ -345,6 +362,35 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
                                help='Path to a ansible_collections directory containing a'
                                ' collection tree to check.')
 
+    validate_tags = subparsers.add_parser(
+        'validate-tags',
+        parents=[build_parser],
+        description="Ensure that collection versions in an Ansible release are tagged"
+                    " in collections' respective git respoitories."
```

Comment:
```suggestion
                    " in collections' respective git repositories."
```

Reply:
Fixed in https://github.com/ansible-community/antsibull/pull/456/commits/edfa6bfd291440b180f13a95c926bc9e11b90a91.